### PR TITLE
Add path to VirtualNode 

### PR DESCRIPTION
**Pull request by EgorBu**
It should help to debug and visualize predictions.

Signed-off-by: egor <egor@sourced.tech>

### CODE REVIEW

File: lookout/style/format/features.py [hunk 1]
```diff
@@ -30,14 +30,15 @@ class FeatureType(enum.Enum):
 
 class VirtualNode:
     def __init__(self,  value: str, start: Position, end: Position,
-                 *, node: bblfsh.Node = None, y: int = None):
+                 *, node: bblfsh.Node = None, y: int = None, path: str = None):
```
Comment (m09) [inline lookout/style/format/features.py]: Should be compulsory argument if we set it everywhere and rely on it later on.

Comment (EgorBu) [reply lookout/style/format/features.py]: TBH - it used to work before without it. I'm using it for analysis - I'm not sure if it should be mandatory

Comment (vmarkovtsev) [reply lookout/style/format/features.py]: @EgorBu Do you mean we do not access this attribute anywhere in the code?

Comment (EgorBu) [reply lookout/style/format/features.py]: Done.

**Pull request merged**